Injector/src/Injector.cpp

```diff
@@ -1,75 +1,90 @@
 #include "Injector.h"
 
-#ifdef _WIN64
-	LPCSTR DllPath = "C:\\Users\\k.rajabzadeh\\source\\repos\\Windows-DLL-Injector\\PayloadDLLBuild\\bin\\Debug\\x64\\PayloadDLL.dll";
-	//LPCSTR DllPath = "C:\\Users\\kourosh\\source\\repos\\WindowsIATHooking\\IATHookingBuild\\bin\\Debug\\x64\\WindowsIATHooking.dll";
-#else
-	LPCSTR DllPath = "C:\\Users\\k.rajabzadeh\\source\\repos\\Windows-DLL-Injector\\PayloadDLLBuild\\bin\\Debug\\Win32\\PayloadDLL.dll";
-	//LPCSTR DllPath = "C:\\Users\\kourosh\\source\\repos\\WindowsIATHooking\\IATHookingBuild\\bin\\Debug\\Win32\\WindowsIATHooking.dll";
-#endif
+#define ProcType_specified
```

Comment:
Define ProcType_specified for use Preset ProcessID

Reply:
Thanks for the pull request
Let me check the changes

Reply:
can you change the section which we get PID inside getpid function to iterate over PIDs for same process name
sometimes there are several processes for a single exe file name (for example 10 process for chrome.exe )
But the ability to iterate over several pids for a process name are removed in that code as I saw
Just copy that section from original code and make it fit with new code and I'll merge the PR. :)